tests/phpunit/tests/dependencies.php

```diff
@@ -148,4 +148,59 @@ public function test_enqueue_before_register() {
 
 		$this->assertContains( 'one', $dep->queue );
 	}
+
+	/**
+	 * Data provider for test_get_etag
+	 *
+	 * @return array
+	 */
+	public function data_provider_get_etag() {
+		return array(
+			'should accept one dependency'              => array(
+				'load'       => array(
+					'abcd' => '1.0.2',
+				),
+				'wp_version' => '',
+				// md5 hash of WP:abcd:1.0.2;'.
+				'expected'   => 'W/"fea2fd8012dd8af0696ebafbaa68db85"',
+			),
+			'should accept empty array of dependencies' => array(
+				'load'       => array(),
+				'wp_version' => '',
+				// md hash of “WP:;”.
+				'expected'   => 'W/"f6457280f73cc597e76df87ee891457a"',
+			),
+			'should accept more then one dependency and wp version' => array(
+				'load'       => array(
+					'abcd' => '1.0.2',
+					'abdy' => '1.0.3',
+				),
+				'wp_version' => '5.4.3',
+				// md hash of “WP:5.4.3;abcd:1.0.2;abdy:1.0.3;”.
+				'expected'   => 'W/"88649143b0142d1491883065e9351178"',
+			),
+		);
+	}
+
+	/**
+	 * Tests get_etag method.
+	 *
+	 * @dataProvider data_provider_get_etag
+	 *
+	 * @param array $load List of scripts to load.
+	 * @param string $wp_version WordPress version.
+	 * @param string $expected Expected etag.
+	 *
+	 * @return void
+	 */
+	public function test_get_etag( $load, $wp_version, $expected ) {
```

Comment:
This test function is only testing the scripts. I recommend renaming it to `test_get_etag_for_scripts` and adding similar function called `test_get_etag_for_styles` that would test the logic using `wp_styles()` and `wp_enqueue_style()`.

Reply:
Thanks for the tests for the `wp_enqueue_style()`. 👍🏽